site/docs/components/table.mdx

```diff
@@ -449,7 +526,38 @@ Vertical lines can be used on tables that have a lot of columns or a lot of simi
 ```
 
 #### React code example:
-_Coming soon to hds-react!_
+```tsx
+const cols = [
+  { key: 'id', headerName: 'Not rendered' },
+  { key: 'firstName', headerName: 'First name' },
+  { key: 'surname', headerName: 'Surname' },
+  {
+    key: 'age',
+    headerName: 'Age',
+    transform: ({ age }) => {
+      return <div style={{ textAlign: 'right' }}>{age}</div>;
+    },
+  },
+  { key: 'profession', headerName: 'Profession' },
+];
+
+const rows = [
+  { id: 1000, firstName: 'Lauri', surname: 'Kekkonen', age: 39, profession: 'Engineer' },
+  // ...
+];
+
+const caption = (
+  <span>
+    <b>Table 5</b>: Units sold by weekday (vertical lines variant)
+  </span>
+);
+
+return (
+  <div style={{ maxWidth: '640px' }}>
+    <Table verticalLines cols={cols} rows={rows} caption={caption} indexKey="id" renderIndexCol={false} />
+  </div>
+);
+```
 
 ### Dense tables
```

Comment:
Do we want to replace the dense also with a react version, like we have done with the other variants?

Reply:
I thought about it but the React version in Storybook was not easily transferrable to Playground. I think it does not matter if this example is in Core for now.

Reply:
True, we can leave it as it is 👍